GETTING_STARTED.md

```diff
@@ -53,7 +53,8 @@ sudo --login --user=postgres psql -c "CREATE USER ofn WITH SUPERUSER CREATEDB PA
 
 This will create the "ofn" user as superuser and allowing it to create databases. If this command fails, check the [troubleshooting section](#creating-the-database) for an alternative.
 
-Next, it is _strongly recommended_ to run the setup script.
+Next, it is _strongly recommended_ to run the setup script. This script assumes your ruby version manager is [rbenv](https://github.com/rbenv/rbenv). Make sure ruby and node versions match the project requirements and run:
```

Comment:
> Make sure ruby and node versions match the project requirements and run: 

I wonder if this is necessary ?  the setup script will check that and try install the correct ruby/node version if needed.